hazelcast/src/test/java/com/hazelcast/internal/ascii/HTTPCommunicator.java

```diff
@@ -426,11 +429,40 @@ public ConnectionResponse headRequestToUndefinedURI() throws IOException {
         return doHead(url);
     }
 
+    public ConnectionResponse getRequestToUndefinedURI() throws IOException {
+        String url = address + "undefined";
+        return doGet(url);
+    }
+
+    public ConnectionResponse postRequestToUndefinedURI() throws IOException {
```

Comment:
are these undefined url methods just for testing purposes?

Reply:
Yes they are.